Solutions/Microsoft 365 Defender/Workbooks/MDO-Workbook.json

```diff
@@ -0,0 +1,1317 @@
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "title": "Summary- MDO threat count",
+        "items": [
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "EmailEvents\r\n| where isnotempty(ThreatTypes)\r\n| count",
+              "size": 3,
+              "title": "Total Threat count ",
+              "timeContextFromParameter": "TimeRange",
+              "exportFieldName": "NetworkMessageIds",
+              "exportParameterName": "NetworkMessageIdsDomain",
+              "exportDefaultValue": "[]",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "tiles",
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "Count",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "greenRed"
+                    }
+                  }
+                ]
+              },
+              "tileSettings": {
+                "titleContent": {
+                  "formatter": 1
+                },
+                "leftContent": {
+                  "columnMatch": "Count",
+                  "formatter": 12,
+                  "formatOptions": {
+                    "palette": "auto"
+                  },
+                  "numberFormat": {
+                    "unit": 17,
+                    "options": {
+                      "maximumSignificantDigits": 3,
+                      "maximumFractionDigits": 2
+                    }
+                  }
+                },
+                "showBorder": false
+              },
+              "chartSettings": {
+                "xAxis": "Count",
+                "group": null,
+                "createOtherGroup": 0
+              },
+              "mapSettings": {
+                "locInfo": "LatLong",
+                "sizeSettings": "Count",
+                "sizeAggregation": "Sum",
+                "legendMetric": "Count",
+                "legendAggregation": "Sum",
+                "itemColorSettings": {
+                  "type": "heatmap",
+                  "colorAggregation": "Sum",
+                  "nodeColorField": "Count",
+                  "heatmapPalette": "greenRed"
+                }
+              }
+            },
+            "customWidth": "25",
+            "name": "query - 0 - Copy - Copy"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "EmailEvents\r\n|where DetectionMethods has_any ('spoof','impersonation')\r\n| project DT=parse_json(DetectionMethods), EmailDirection, SenderFromAddress, AdditionalFields \r\n| evaluate bag_unpack(DT) \r\n| summarize count()by tostring(Phish[0]) ",
+              "size": 0,
```

Comment:
Can we change the height to Tiny. The other two queries in the same line are at the same height.

Just a suggestion, if deemed appropriate, let's include all the count KQLs in a single query.

Reply:
completed